docker/Dockerfile.build.aarch64-linux

```diff
@@ -94,6 +94,26 @@ RUN LIBTIFF_VERSION=4.1.0 && \
     cd && \
     rm -rf /tmp/tiff-${LIBTIFF_VERSION}
 
+# OpenJPEG
+RUN OPENJPEG_VERSION=2.3.1 && \
+    cd /tmp && \
+    curl -L https://github.com/uclouvain/openjpeg/archive/v${OPENJPEG_VERSION}.tar.gz | tar -xzf - && \
+    cd openjpeg-${OPENJPEG_VERSION} && mkdir build && cd build && \
+    echo "set(CMAKE_SYSTEM_NAME  Linux)" > toolchain.cmake && \
+    echo "set(CMAKE_SYSTEM_PROCESSOR aarch64)" >> toolchain.cmake && \
+    echo "set(CMAKE_C_COMPILER aarch64-linux-gnu-gcc)" >> toolchain.cmake && \
+      CFLAGS="-fPIC" \
+      CXXFLAGS="-fPIC" \
+      CC=aarch64-linux-gnu-gcc \
+      CXX=aarch64-linux-gnu-g++ \
+    cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=toolchain.cmake -DCMAKE_INSTALL_PREFIX=/usr/aarch64-linux-gnu/ .. && \
```

Comment:
I think `-DCMAKE_BUILD_TYPE=Release` is the default, but it won't do any harm to set is explicitly.